backend/crypt/crypt.go

```diff
@@ -151,6 +151,14 @@ length and if it's case sensitive.`,
 				},
 			},
 			Advanced: true,
+		}, {
+			Name: "custom_suffix",
+			Help: `If this is set it will override the default suffix of ".bin".
+
+If you leave blank, or do not set to a valid extension, e.g. "jpg" instead of ".jpg", 
+then the suffix will be empty. This may be useful when the path length is critical.`,
```

Comment:
I'm not sure leaving it blank will work. I think if the user leaves it blank then they will get the default won't they?

I guess there might be a difference between not providing the `custom_suffix` argument and providing `custom_suffix =`.

The point that no suffix is useful when the path length is critical is a very good one.

Reply:
have updated as per your suggestion, e.g. ```none``` now sets the suffix to blank.